front/src/WebRtc/DiscussionManager.ts

```diff
@@ -104,11 +104,11 @@ export class DiscussionManager {
     }
 
     public addParticipant(
-        userId: number|string,
+        userId: number|'me',
```

Comment:
maybe the userId could be -1 instead of 'me'?

Reply:
I'd rather keep "me" because it is a completely different type, so the Typescript compiler can tell us if we don't handle that case.